google-cloud-clients/google-cloud-webrisk/pom.xml

```diff
@@ -34,49 +30,11 @@
       <groupId>com.google.api.grpc</groupId>
       <artifactId>grpc-google-cloud-webrisk-v1beta1</artifactId>
     </dependency>
-    <dependency>
-      <groupId>io.grpc</groupId>
-      <artifactId>grpc-netty-shaded</artifactId>
-    </dependency>
-    <dependency>
-      <groupId>io.grpc</groupId>
-      <artifactId>grpc-stub</artifactId>
-    </dependency>
-    <dependency>
-      <groupId>io.grpc</groupId>
-      <artifactId>grpc-auth</artifactId>
-    </dependency>
-    <dependency>
-      <groupId>${project.groupId}</groupId>
-      <artifactId>google-cloud-core</artifactId>
-      <type>test-jar</type>
-      <scope>test</scope>
-    </dependency>
     <dependency>
       <groupId>junit</groupId>
       <artifactId>junit</artifactId>
       <scope>test</scope>
     </dependency>
-    <dependency>
-      <groupId>org.easymock</groupId>
-      <artifactId>easymock</artifactId>
-      <scope>test</scope>
-    </dependency>
-    <dependency>
-      <groupId>org.objenesis</groupId>
-      <artifactId>objenesis</artifactId>
-      <scope>test</scope>
-    </dependency>
-    <dependency>
-      <groupId>com.google.truth</groupId>
-      <artifactId>truth</artifactId>
```

Comment:
Have you run the build after this?

Reply:
Yes i run **mvn clean install -DskipITs**  after removing this and its working fine.

Reply:
I think you have to run with test case because this is test related dependencies.. we don't want any runtime surprises

Reply:
i tried with ""mvn clean install"" and all the test cases are run successfully.